### ISSUE TITLE
[FR] Collapsible infobox without onhover movement

### ISSUE THREAD

**Issue by scottTomaszewski** [SlRvb/Obsidian--ITS-Theme#127]
### Is your feature request related to a problem? Please describe.
I want to have a collapsible infobox (`> [!infobox]+`) that doesnt shift things down when the cursor hovers as it makes any links hard to click

### Describe the solution you'd like
Ideally, I want the collapsing icon to always show, or any other solution that avoids elements from moving when hovering over the infobox

### Describe alternatives you've considered
I looked at modifying the css: 
```
.callout[data-callout~=infobox]:not(:hover):not(.is-collapsed) .callout-title, .callout[data-callout~=infobox] .callout-title :is(.callout-icon, .callout-title-inner) {
    display: none;
}
```
but removing the `display:none` renders things slightly differently, and changing it to a different `display` also doesnt quite format correctly

**Comment by SlRvb** [comment]
I think I've applied a fix to this to stop the twitching movement 3d9f987. Update the theme/snippet and let me know if this fix is alright for you?

**Comment by scottTomaszewski** [comment]
Yes!  this is perfect!  Thank you so much!
